packages/terra-form-select/src/Combobox.jsx

```diff
@@ -88,7 +88,7 @@ const propTypes = {
   /**
    * The selected value.
    */
-  value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+  value: PropTypes.oneOfType([PropTypes.string, PropTypes.number, PropTypes.array]),
```

Comment:
nit: the props table generated on the site does not show this information well (it says that the type is a ‘union’). I’d explicitly call this piece out in the inline comment above so that the consumers are aware of what types this prop can be when they look up documentation.

Reply:
fixed: https://github.com/cerner/terra-core/pull/2646/commits/734f0b26bb3a546fcbbdbbd1babc37bd9461b17c